splunk_connect_for_snmp/traps.py

```diff
@@ -174,18 +174,19 @@ def main():
         cbCtx=observer_context,
     )
 
-    # UDP over IPv4, first listening interface/port
-    config.addTransport(
-        snmp_engine,
-        udp.domainName,
-        udp.UdpTransport().openServerMode(("0.0.0.0", 2162)),
-    )
-
+    # UDP socket over IPv6 listens also for IPv4
     if IPv6_ENABLED:
         config.addTransport(
             snmp_engine,
             udp6.domainName,
-            udp6.Udp6Transport().openServerMode(("::", 2163)),
```

Comment:
Probably we can listen both ports 2162 and 2163, **just for backward compatibility**. It would be undocumented.  Because  when users will update SC4SNMP to latest version they will have a problem.